tests/usb_phy_shim.py

```diff
@@ -49,12 +49,12 @@ def run(self):
 
                     #sample TXV for new packet
                     if xsi.sample_port_pins(self._txv) == 1:
-                        print "Receiving packet {}".format(i)
```

Comment:
Didn't we already convert this stuff to python3??

Reply:
Oh.. I forget to convert that one. Now's OK